src/credentials/DeviceAttestationVerifier.cpp

```diff
@@ -52,6 +52,21 @@ class UnimplementedDACVerifier : public DeviceAttestationVerifier
         (void) certDeclBuffer;
         return AttestationVerificationResult::kNotImplemented;
     }
+
+    AttestationVerificationResult ValidateCertificateDeclarationPayload(const ByteSpan & certDeclBuffer,
+                                                                        const ByteSpan & firmwareInfo, uint16_t clusterVendorId,
+                                                                        uint16_t clusterProductId, uint16_t dacVendorId,
+                                                                        uint16_t dacProductId, uint16_t paiProductId) override
+    {
+        (void) certDeclBuffer;
+        (void) firmwareInfo;
+        (void) clusterVendorId;
+        (void) clusterProductId;
+        (void) dacVendorId;
+        (void) dacProductId;
+        (void) paiProductId;
```

Comment:
There is a need to validate PAI's PID as well, and we should add paaVendorId for future proofing checks, since the PAA cert may contain a VID.

Reply:
We are validating paiProductId.

Reply:
I guess PID is never included in PAA.